examples/validators/policy-controller-validate/invalid-configmap/.expected/results.yaml

```diff
@@ -0,0 +1,12 @@
+name: policy-controller-validate
+items:
+- message: |+
+    Found 1 violations:
+
+    [1] The following banned keys are being used in the config map: {"private_key"}
```

Comment:
The only non-structured content that needs to be part of the message are the following:

- `The following banned keys are being used in the config map: {"private_key"}`
- violatedConstraint

Other metadata should be conveyed using other fields here:
- name
- path

Number of violations is an emergent property of the function results.